src/main/java/task/DataGenerator.java

```diff
@@ -0,0 +1,31 @@
+package task;
+
+import com.github.javafaker.Faker;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class DataGenerator {
+
+    private DataGenerator() {
```

Comment:
whats the purpose of this class if it have private constructor?
Only to hold 1 static method?

Reply:
Fixed